scripts/quick-install-ubuntu.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail  ## https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/
+
+# This script assumes root permissions on a fresh Ubuntu Digital Ocean droplet
+# with these recommended specs: 16 GB Memory / 8 Intel vCPUs / 320 GB Disk
+
+# dismiss the popups
+export DEBIAN_FRONTEND=noninteractive 
+
+apt update -y && apt upgrade -y
+
+apt install docker jq -y
```

Comment:
I would have thought this would get you an old version of Docker (comes from the distro package repo). We want a recent version of Docker, which afaik requires defining the DockerCo package repo then installing from there.

Reply:
updated. `apt install docker.io` uses a current version